rasa/nlu/extractors/extractor.py

```diff
@@ -40,7 +46,7 @@ class EntityTagSpec(NamedTuple):
     num_tags: int
 
 
-class EntityExtractor(Component):
+class EntityExtractorGraphComponent(NLUGraphComponent, abc.ABC):
```

Comment:
`EntityExtractor` is really just a collection of functions. It could probably just be a module tbh. To make the conversion easier I can just make it into a mixin, i.e. remove it's superclasses, so it's just used for other classes to gain this functionality.